platform/accessibility/common-issues/pdf/table-missing-headers.md

```diff
@@ -0,0 +1,75 @@
+# [PDF]: Table is missing headers
+
+## Table of contents
+
+  - [Descrption of the issue](#description-of-the-issue)
+  - [Definition of done](#definition-of-done)
+  - [Steps to fix the issue](#steps-to-fix-the-issue)
+    - [Define table headers in Microsoft Word](#define-table-headers-in-microsoft-word)
+    - [Define table headers in Adobe Acrobat Pro](#define-table-headers-in-adobe-acrobat-pro)
+  - [Test for compliance](#test-for-compliance)
+    - [Microsoft Word accessibility panel](#microsoft-word-accessibility-panel)
+    - [Acrobat Pro accessibility tools](#acrobat-pro-accessibility-tools)
+  - [Learn more](#learn-more)
+
+## Description of the issue
+
+- Synopsis: Make tables accessible in PDF documents by using table elements. The applicable standard from Section508.22.h dictates to "associate data and header cells for multi-dimensional data tables."
+- Severity: [508-defect-1❗️Launch blocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/1Copenut-pdfs-table-header-missing/platform/accessibility/guidance/defect-severity-rubric.md#types-of-severity-1-issues-in-pdf-documents) for new documents. Documents previously published should be remediated as soon as possible.
+- Deque University help link: https://dequeuniversity.com/rules/comply-pdf/1.0/table-miss-header
+- WCAG Success Criteria: [1.3.1 Info and Relationships](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
+
+## Definition of done
+
+- There SHOULD be **0 table header violations** in the Microsoft Word [Check Accessibility panel](https://support.microsoft.com/en-us/office/improve-accessibility-with-the-accessibility-checker-a16f6de0-2f39-4a2b-8bd8-5ad801426c7f)
+- There MUST be **0 table header violations** in the Acrobat [Accessibility Checker](https://www.adobe.com/accessibility/products/acrobat/using-acrobat-pro-accessibility-checker.html)
+- Consider exporting an accessibility report from Adobe Acrobat while running the Accessibility Checker
+
+## Steps to fix the issue
+
+Tables that are missing table headers can be remediated in Microsoft Word (preferred) or Adobe Acrobat.
+
+### Define table headers in Microsoft Word
+
+Table header rows can be defined in a few steps:
```

Comment:
I just realized I missed this before, but we do have recommendations in the style guide about [formatting multi-action procedures](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/documentation/style-guide/formatting-and-organization.md#procedures).

The main difference is to use **bold** instead of `code` We also try to avoid saying "Click" in favor of "Select"

I wouldn't delay publishing based on this, though, just a note for the future.

Reply:
I went ahead and updated them, so I have a good guide for the next 6-7 documents. Thank you @gmaciolek !

Reply:
🎉  go team!